.packit.yaml

```diff
@@ -1,47 +1,61 @@
 specfile_path: dist/beakerlib.spec
-#synced_files:
-#    - beakerlib.spec
+
+files_to_sync:
+  - src: dist/*
+    dest: .
+  - .packit.yaml
```

Comment:
I guess you also want to sync specfile?

Reply:
the spec file is in the dist directory, so it should be covered

Reply:
So, you're not building (or planning to build) `rpms` for pull requests?

Reply:
I do not think it is necessary to build on downstream PR while we build on the upstream PR already. Respectively, the build will be triggered based on the merge of the PR.

Reply:
I mean, the spec file is present in the upstream `/dist` directory so it gets synced to the downstream from there. And while have the same spec, I consider the upstream CI sufficient.

Reply:
Ah, I see! I somehow haven't noticed the `dist` directory. I thought you were speaking about the dist git repo. Now everything makes sense ;)